cmd/exec.go

```diff
@@ -107,25 +106,21 @@ func executeExec(ctx context.Context, dev *model.Dev, args []string) error {
 		dev.Container = p.Spec.Containers[0].Name
 	}
 
-	if dev.RemoteModeEnabled() {
```

Comment:
can we leave a flag (or env var) to enable the old behavior for now? It would be good to check with some of the heavy users to see if they have any issues or concerns with this shift.